itest/rust/src/builtin_tests/containers/signal_test.rs

```diff
@@ -300,8 +273,8 @@ fn signal_symbols_engine_inherited(ctx: &crate::framework::TestContext) {
     // Add to tree, so signals are propagated.
     ctx.scene_tree.clone().add_child(&node);
 
-    let mut sig = node.signals().renamed();
-    sig.connect_self(|this: &mut Emitter| {
```

Comment:
The original thing that motivated the PR... This is still very nice to see! :muscle: